instrumentation/opentelemetry-instrumentation-starlette/src/opentelemetry/instrumentation/starlette/__init__.py

```diff
@@ -278,14 +292,14 @@ def _uninstrument(self, **kwargs):
 
 
 class _InstrumentedStarlette(applications.Starlette):
-    _tracer_provider = None
-    _meter_provider = None
+    _tracer_provider: TracerProvider | None = None
+    _meter_provider: MeterProvider | None = None
     _server_request_hook: ServerRequestHook = None
     _client_request_hook: ClientRequestHook = None
     _client_response_hook: ClientResponseHook = None
-    _instrumented_starlette_apps = set()
+    _instrumented_starlette_apps: set[applications.Starlette] = set()
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args: Any, **kwargs: Any):
```

Comment:
I'm trying to find something better here: https://github.com/python/typing/discussions/1889